cmd/params/params.go

```diff
@@ -470,16 +466,16 @@ func loadSanitizeParams(v *viper.Viper) (SanitizeParams, error) {
 		)
 	}
 
-	params.HideFileNames = hideFileNamesPatterns
-
-	return params, nil
+	return SanitizeParams{
+		HideBranchNames:     hideBranchNamesPatterns,
+		HideFileNames:       hideFileNamesPatterns,
+		HideProjectFolder:   vipertools.FirstNonEmptyBool(v, "relative-paths", "settings.relative_paths"),
```

Comment:
Should we rename this to `settings.hide_project_folder`?

Reply:
makes more sense to be explicitly defined.

Reply:
So we leave it unchanged (`relative_paths`)?

Reply:
`hide_project_folder` is more meanful IMO

Reply:
Yep, I agree. So `--hide-project-folder` and `settings.hide_project_folder = true`.